spi/src/main/java/org/opensearch/jobscheduler/spi/LockModel.java

```diff
@@ -80,6 +80,8 @@ public LockModel(String jobIndexName, String jobId, Instant lockTime,
                      long lockDurationSeconds, boolean released, long seqNo, long primaryTerm) {
         this.lockId = jobIndexName + LOCK_ID_DELIMITR + jobId;
         this.jobIndexName = jobIndexName;
+        // The jobId parameter does not necessarily need to represent the id of a job scheduler job, as it is being used
```

Comment:
may be create an issue in repo to rename this completely to just lockId everywhere? so the paradigm is changed in job scheduler - cleaning up stale locks and any other refactor needed with this change

Reply:
Done!